### PR TITLE
Generate static TfLiteEvalTensors

### PR DESCRIPTION
**Pull request by rascani**
This PR generates all of the TfLiteEvalTensors for the graph. It also generates all of the static buffers used for tensors that were present in the flatbuffer.

BUG=b/295077140

### CODE REVIEW

File: codegen/utils.py [hunk 1]
```diff
@@ -30,6 +31,17 @@ def bool_to_c_str(b: bool) -> str:
...
 
+def split_into_chunks(
+    data: Iterable[Any],
+    chunk_size: int) -> Generator[Tuple[Any, ...], None, None]:
+  """Splits an iterable into chunks of a given size."""
+  data_iterator = iter(data)
+  chunk = tuple(itertools.islice(data_iterator, chunk_size))
+  while chunk:
+    yield chunk
+    chunk = tuple(itertools.islice(data_iterator, chunk_size))
```
Comment (advaitjain) [inline codegen/utils.py:42]: this reads a bit funny.

Is there any way to write this where the loop looks something like:
```py
while <not sure what the condition would be>:
    chunk = tuple(itertools.islice(data_iterator, chunk_size))
    yield chunk
```

**Review by advaitjain — CHANGES_REQUESTED** [review]
one nit question. Other than that lgtm.

File: codegen/utils.py (hunk 1 above)
Comment (rascani) [reply codegen/utils.py:42]: WDYT of this:

```
while True:
  chunk = tuple(itertools.islice(data_iterator, chunk_size))
  if not chunk:
    break
  yield chunk
```

Comment (advaitjain) [reply codegen/utils.py:42]: +1. that seems  more readable to me.

Comment (rascani) [reply codegen/utils.py]: Done

**Review by advaitjain — APPROVED** [review]

**Pull request merged**